course7/homeworks/mathias/complex_rpn/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+project(complex_rpn VERSION 1.0 LANGUAGES CXX)
+
+if(MSVC)
+    add_compile_options("/permissive-" "/W4" "/WX" "$<$<CONFIG:RELEASE>:/O2>")
+else()
+    add_compile_options("-std=c++14" "-Wall" "-Wextra" "-Werror" "$<$<CONFIG:RELEASE>:-O3>")
+endif()
+
+add_executable(complex_rpn main.cpp complex.h complex.cpp rpn_calculator.h rpn_calculator.cpp)
+target_include_directories(complex_rpn PRIVATE ${Boost_INCLUDE_DIR})
```

Comment:
Boost_INCLUDE_DIR is not going to be populated, if there is no find_package(Boost ...)